compiler/modules/nback/scaffold.bal

```diff
@@ -106,8 +106,16 @@ type ModuleDI record {|
     DIFile[] files;
     DICompileUnit compileUnit;
     DISubroutineType funcType;
+    boolean debugFull;
 |};
 
+// Debug location will always be added
+public const int ERROR_CONSTRUCTOR = 0; // only include file name not position
+public const int CALL_INSTRUCTION = 1;
+// Debug locattion for these will be added only in full debug
+public const int OTHER_INSTRUCTION = 2;
+
+public type DebugLocationTarget ERROR_CONSTRUCTOR|CALL_INSTRUCTION|OTHER_INSTRUCTION;
```

Comment:
It's not the `Target`. Maybe `Origin`.

The names of the constants and names of the type need to be harmonious and the 3 names need to be consistent. INSTRUCTION doesn't add value: everything comes from an instruction.

Reply:
fixed